### PR TITLE
Add log_dir to ToolAttachWithRL.srv.

### PR DESCRIPTION
**Pull request by arkokundu**
Background
Add `string log_dir` to ToolAttachWithRL.srv.
We want rltc to know the log directory when cmd attach is called for tool_change, so that the rollout data can be saved in the right action log directory. 
What's new

❌ bug fixes
❌ New feature is accompanied by new test: [name and description of new test].
Related work

❌ This PR needs [link]
❌ [link] needs this PR
TODOs / Nice-To-Haves 

❌ [Add new unit test for new feature.]
❌ [Add new system test for new feature.]
Tests

✅ Tested:
Quality

✅ New code is purely Python 3.8 and up.
✅ New code passed all 24 pre-commit hooks including unit tests.
Note to reviewers

### CODE REVIEW

File: srv/ToolAttachWithRL.srv [hunk 1]
```diff
@@ -1,4 +1,5 @@
 string tool_id
+string date_time
```
Comment (duanyutong) [inline srv/ToolAttachWithRL.srv:2]: `log_dir` is probably better

File: srv/ToolAttachWithRL.srv [hunk 2]
```diff
@@ -1,5 +1,5 @@
 string tool_id
-string date_time
+string log_dir
 ---
 bool evaluate
-string message
+string message
```
Comment (duanyutong) [inline srv/ToolAttachWithRL.srv:5]: revert this change. this violates EOL standards

**Comment by arkokundu** [comment]
This PR is ready for review now.

**Review by duanyutong — COMMENTED** [review]
please update PR title

**Review by duanyutong — APPROVED** [review]
